src/test/extension.test.ts

```diff
@@ -1,15 +1,100 @@
+    'sentry.decorations.inline': false,
+    'sentry.organization': 'sourcegraph',
+    'sentry.projects': [
+        {
+            additionalProperties: {
+                contentText: 'View sourcegraph/sourcegraph_dot_com errors',
+                hoverMessage: 'View errors matching "$1" in Sentry',
+                query: '$1',
+            },
+            name: 'Webapp typescript errors',
+            patternProperties: {
+                fileMatches: [/(web|shared|src)\/.*\.tsx?/, /\/.*\\.ts?/],
+                lineMatches: [
+                    /throw new Error+\(['"]([^'"]+)['"]\)/,
+                    /console\.(warn|debug|info|error|log)\(['"`]([^'"`]+)['"`]\)/,
+                    /log\.(Printf|Print|Println)\(['"]([^'"]+)['"]\)/,
+                ],
+                repoMatches: [/sourcegraph\/sourcegraph/, /bucket/],
+            },
+            projectId: '1334031',
+        },
+        {
+            additionalProperties: {
+                contentText: 'View sourcegraph/dev-repo errors',
+                hoverMessage: 'View errors matching "$1" in Sentry',
+                query: '$1',
+            },
+            name: 'Dev env errors',
+            patternProperties: {
+                fileMatches: [/(dev)\/.*\\.go?/],
+                lineMatches: [/log\.(Printf|Print|Println)\(['"]([^'"]+)['"]\)/],
+                repoMatches: [/dev-repo/],
+            },
+            projectId: '213332',
+        },
+    ],
+}
+
+export let projects: SentryProject[] = [
+    {
+        name: 'Webapp typescript errors',
+        projectId: '1334031',
+        patternProperties: {
+            repoMatches: [/sourcegraph\/sourcegraph/, /bucket/],
+            fileMatches: [/(web|shared|src)\/.*\.tsx?/, /\/.*\\.ts?/],
+            lineMatches: [
+                /throw new Error+\(['"]([^'"]+)['"]\)/,
+                /console\.(warn|debug|info|error|log)\(['"`]([^'"`]+)['"`]\)/,
+                /log\.(Printf|Print|Println)\(['"]([^'"]+)['"]\)/,
+            ],
+        },
+        additionalProperties: {
+            contentText: 'View sourcegraph/sourcegraph_dot_com errors',
+            hoverMessage: 'View errors matching "$1" in Sentry',
+            query: '$1',
+        },
+    },
+
+    {
+        name: 'Dev env errors',
+        projectId: '213332',
+        patternProperties: {
+            repoMatches: [/dev-repo/],
+            fileMatches: [/(dev)\/.*\\.go?/],
+            lineMatches: [/log\.(Printf|Print|Println)\(['"]([^'"]+)['"]\)/],
+        },
+        additionalProperties: {
+            contentText: 'View sourcegraph/dev-repo errors',
+            hoverMessage: 'View errors matching "$1" in Sentry',
+            query: '$1',
+        },
+    },
+]
+
+describe('check for configurations', () => {
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
```

Comment:
This function is duplicated multiple times. If you want this to execute before every test in this file, wrap all describe blocks in another describe block and apply the beforeEach there. If you only need it for _most_ tests, save the function in a variable and pass that to `beforeEach`. If you want this to run before every test in every file, you could define it globally in a `beforeEach.test.ts` or share it as a function